File: apprefurbished/src/main/java/com/androidcollider/easyfin/faq/FAQMVP.java
package com.androidcollider.easyfin.faq;

import android.util.Pair;

import java.util.List;

/**
 * @author Ihor Bilous
 */

public interface FAQMVP {

    interface Model {

        List<Pair<String, String>> getInfo();
    }

    interface View {

        void setInfo(List<Pair<String, String>> list);
    }

    interface Presenter {

        void setView(FAQMVP.View view);

        void loadInfo();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/faq/FAQMVP.kt
package com.androidcollider.easyfin.faq

import android.util.Pair

/**
 * @author Ihor Bilous
 */
interface FAQMVP {
    interface Model {
        val info: List<Pair<String, String>>
    }

    interface View {
        fun setInfo(list: List<Pair<String, String>>)
    }

    interface Presenter {
        fun setView(view: View?)
        fun loadInfo()
    }
}